theme/snippets/product-images-slider.liquid

```diff
@@ -0,0 +1,11 @@
+<div class="ProductImagesSlider splide">
+	<div class="splide__track">
+		<ul class="splide__list">
+      {% for image in product.images %}
+        <li class="splide__slide py6 md:py3 pr2 md:pr4">
+          <img class="ProductImagesSlider__img w100 h100 fit-contain" src="{{ image | img_url: 'master' }}" alt="{{ image.alt }}" />
+        </li>
+      {% endfor %}
+		</ul>
+	</div>
+</div>
```

Comment:
formatting looks weird here but it is fine in VS!

Reply:
Yeah, formatting is weird. Almost looks like this file is using tabs over spaces?